UseCase_I.ipynb

```diff
@@ -372,7 +372,8 @@
    "source": [
     "for i in range(len(galex_image_table)):\n",
     "    if (('image/fits' in galex_image_table[i].format) and\n",
-    "        (galex_image_table['enrValue'][i]==2.35e-07)):\n",
+    "        (galex_image_table['energy_bounds_center'][i]==2.35e-07) and\n",
+    "        (galex_image_table[i]['productType'] == 'SCIENCE')):\n",
```

Comment:
Nitpicking to make it consistent for the two constraints.

OTOH, using `==` for numbers is both asking for trouble and teaching a very bad practice in case people where to use it for floats (is that column an int or a float column anyway), so, separately from this PR we may need to revise notebooks to get rid of those.

```suggestion
    "        (galex_image_table['productType'][i] == 'SCIENCE')):\n",
```